libs/src/services/creatorNode/index.js

```diff
@@ -137,19 +137,22 @@ class CreatorNode {
    * Uploads creator content to a creator node
    * @param {object} metadata the creator metadata
    */
-  async uploadCreatorContent (metadata) {
+  async uploadCreatorContent (metadata, blockNumber = null) {
```

Comment:
are there other routes where this might be useful?

Reply:
yeah there most likely are but i figured we could fix only what needed to be fixed in this PR. i'm open to making any other changes but only these are susceptible to the race condition (aka these are the only two places that updateCreatorNodeEndpoint) is called

Reply:
this cool? @hareeshnagaraj @dmanjunath

Reply:
yes def not worth pre-empting this